elements/pfe-jump-links/src/pfe-jump-links-nav.scss

```diff
@@ -0,0 +1,324 @@
+  --pfe-jump-links--BackgroundColor: BackgroundColor;
+}
+
+::slotted([slot="heading"]) {
+  margin: 0 0 1rem 0 !important;
+  margin-bottom: calc(#{pfe-var(container-spacer)} * 1);
+  font-size: 14px;
+  font-weight: 400;
+  text-transform: uppercase;
+}
+
+ul{
+  &.pfe-jump-links-nav {
+    &[horizontal] {
+
+    }
+    
+    padding: 1rem 1rem 1rem 0;
+    padding: 1rem 1rem 1rem 0;
+    margin: 0;
+    list-style: none;
+  }
+  &.sub-nav {
+    margin: 0;
+    padding: 0;
+    overflow-y: hidden;
+    transition: flex 1s linear;
+    display: flex;
+    flex-direction: column;
+    flex: 0;
+  }
+}
+
+li {
+  border-left: 3px solid transparent;
+  display: block;
+  text-decoration: none;
+}
+
+li.expand {
+  ul.sub-nav {
+    flex: 1;
+  }
+  a.sub-section {
+    display: table;
+    height: auto;
+  }
+}
+
+a {
+  display: table;
+  text-decoration: none;
+  font-size: 16px;
+  padding: 8px;
+  line-height: 1.5;
+  border-left: 3px solid transparent;
+  position: relative;
+  &.sub-section{
+    display: none;
+    // height: 0;
+    margin-left: 12px;
+    font-size: 14px;
+  }
+  &::after {
+    content: "";
+    position: absolute;
+    width: calc(100% - 4px);
+    height: calc(100% - 4px);
+    top: 0;
+    left: 0;
+  }
+  &:focus {
+    outline: none;
+
+    &::after {
+      border-radius: 3px;
+      border: 2px solid #62A1F4;
+    }
+  }
+}
+
+:host([default]) {
+  border: 1px solid #D2D2D2;
+  ::slotted([slot="heading"]) {
+    color: #151515;
+  } 
+  ul.pfe-jump-links-nav {
+    border-left: 1px solid #D2D2D2;
+  }
+
+  li{
+    &[active] {
+      border-left: 3px solid #0066CC;
+    }
+  }
+
+  a {
+    color: #72767B;
+    position: relative;
+
+    &:hover {
+      color: #151515;
+    }
+    &[active] {
+      color: #151515;
+    }
+    &::after {
+      content: "";
+      position: absolute;
+      width: calc(100% - 4px);
+      height: calc(100% - 4px);
+      top: 0;
+      left: 0;
+    }
+
+    &:focus {
+      outline: none;
+
+      &::after {
+        border-radius: 3px;
+        border: 2px solid #62A1F4;
+      }
+    }
+  }
+}
+
+:host([pfe-color="darkest"]) {
+  ::slotted([slot="heading"]) {
+    color: #CCCCCC;
+  } 
+  background-color: #151515;
+  border: 1px solid #72767B;
+  ul.pfe-jump-links-nav {
+    border-left: 1px solid #72767B;
+  }
+  li{
+    &[active] {
+      border-left: 3px solid #EE0000;
+    }
+  }
+  a {
+    color: #CCCCCC;
+    &:hover {
+      color: darken($color: #CCCCCC, $amount: 20);
+    }
+    &[active] {
+      color: #fff;
+    }
+
+    &::after {
+      content: "";
+      position: absolute;
+      width: calc(100% - 4px);
+      height: calc(100% - 4px);
+      top: 0;
+      left: 0;
+    }
+
+    &:focus {
+      outline: none;
+
+      &::after {
+        border-radius: 3px;
+        border: 2px solid #62A1F4;
+      }
+    }
+  }
+}
+
+:host([hidden]) {
+  display: none;
+}
+
+:host([horizontal]) {
+  
+  nav {
+    background-color: #F0F0F0;
```

Comment:
suggested, using   surface--lightest` or `surface--lighter` and then tweaking with the theme layer if needed?

Preview all colors here: http://localhost:8000/elements/pfe-styles/demo/